docs/tutorial.md

```diff
@@ -0,0 +1,71 @@
+# Tutorial: First 15 Minutes
+This tutorial walks you through setting up Tilt for your project. It should take 15 minutes, and assumes you've already [installed Tilt](install.html). Before you begin, you may want to join the `#tilt` channel in [Kubernetes Slack](http://slack.k8s.io) for technical and moral support.
+
+Start by `cd`'ing into a project you can already build and deploy.
+
+## Example Tiltfile
+At the end of this guide, your Tiltfile will look something like this:
+```python
+# Deploy: tell Tilt what yaml to deploy
+k8s_yaml('app.yaml')
+
+# Build: tell Tilt what images to build from which directories
+docker_build('companyname/frontend', 'frontend')
+docker_build('companyname/backend', 'backend')
+# ...
+
+# Watch: tell Tilt how to connect locally (optional)
+k8s_resource('frontend', port_forwards=8080)
+```
+
+## Hello World
+Run `tilt up` to enter Tilt's Heads-Up Display. Instead of writing your configuration all at once, we'll use Tilt interactively. Each time you save your configuration, Tilt will reexecute it. Tilt should be complaining there's no file named `Tiltfile`. Open it in your editor and write:
```

Comment:
```suggestion
Run `tilt up` to enter Tilt's Heads-Up Display. Instead of writing your configuration all at once, we'll use Tilt interactively. Each time you save your configuration, Tilt will reexecute it.

Right now, Tilt should be complaining there's no file named `Tiltfile`. Open it in your editor and write:
```